### PR TITLE
We should not change objects passed to our API (fix #7)

### LINKED ISSUE #7: The createWorkers function clears the browsers argument object
https://www.devexpress.com/support/center/Question/Details/T341284

### PR DESCRIPTION
**Pull request by LavrovArtem**
\cc @inikulin @AlexanderMoskovkin

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -8,64 +9,44 @@ function generateWorkerName(browser) {
...
-                createWorkers(browsers, callback, workerIds);
-            } else
-                callback && callback(workerIds, err);
+        bsClient.createWorker(browserInfo, function(err, worker) {
+            callback(err, !err && worker.id);
         });
-    } else
-        callback && callback(workerIds);
+    }, function(err, workerIds) {
+        callback && callback(workerIds, err);
```
Comment (AlexanderMoskovkin) [inline index.js]: It's a strange for me that we returns `err` as the second `callback` argument and `workerIds` as the first. Usually, it's `callback(err, res)`. But I see, it was before this commit. If we will change this it's a breaking change. /cc @inikulin 

lgtm for this commit changes

**Comment by inikulin** [comment]
lgtm

**Comment by LavrovArtem** [comment]
We need to publish new version

**Comment by AlexanderMoskovkin** [comment]
just @churkin can do it

**Pull request merged**